README.md

```diff
@@ -35,6 +35,29 @@ vexml.Vexml.fromMusicXML(musicXML).render({ element: div, width: width });
 
 This will render a child SVG element whose height will automatically adjust to fit the container. There is currently no option to disable this.
 
+## Motivation
+
+> [!NOTE]
+> At the time of writing, most of these features are aspirational. Please see the [Usage](#Usage) section for a list of available features.
```

Comment:
Could be helpful to call out its distinct advantages over OSMD

Reply:
I'll hold off for comparisons until the library is published. I think it might be misleading to compare aspirations to actual features in other libraries.